src/components/figures/chart/chartJs/chartjs_show_values_plugin.ts

```diff
@@ -2,13 +2,15 @@ import {
   chartFontColor,
   isTrendLineAxis,
 } from "@odoo/o-spreadsheet-engine/helpers/figures/charts/chart_common";
+import { ChartType as AllChartType } from "@odoo/o-spreadsheet-engine/types/chart";
 import { ChartMeta, ChartType, Plugin } from "chart.js";
 import { computeTextWidth } from "../../../../helpers";
 import { Color } from "../../../../types";
 
 export interface ChartShowValuesPluginOptions {
+  type: AllChartType;
   showValues: boolean;
-  background?: Color;
+  background?: (value: number | string, dataset: ChartMeta, index: number) => Color | undefined;
```

Comment:
```suggestion
  background: (value: number | string, dataset: ChartMeta, index: number) => Color | undefined;
```
Since you now provide a default callback in getChartShowValues
the optional chaining should also be removed everywhere in the code